src/lib/utils.ts

```diff
@@ -19,3 +20,32 @@ export const formatDate = (iso: string | undefined) => {
     return iso;
   }
 };
+
+export const getAuthToken = async (): Promise<string | null> => {
```

Comment:
I think we missed to handle `null` checks here. Just in case to be on the safer side.